config/locales/custom/nn/activerecord.yml

```diff
@@ -0,0 +1,10 @@
+nn:
+  activerecord:
+    attributes:
+      poll/question:
+        mandatory_answer: "Obligatorisk svar"
```

Comment:
Could you remind me where we got these texts from? I've forgotten :sweat_smile:.

Reply:
I've used an online translator 😄 . I'd ask the customer for verification.